azure-communication-ui/azure-communication-ui/src/main/res/values/azure_communication_ui_strings.xml

```diff
@@ -56,5 +56,6 @@
     <string name="azure_communication_ui_snack_bar_dismiss">Dismiss</string>
     <string name="azure_communication_ui_cal_state_error_call_end">You were removed from the call due to an error.</string>
     <string name="azure_communication_ui_cal_state_error_call_join">Unable to join the call due to an error.</string>
+    <string name="azure_communication_ui_service_notification_title">Ongoing call</string>
```

Comment:
Muhammad, Please check with UI team Alex to confirm this message is good.

Reply:
Checked with Alex, he said to do something like this:

Call · 3 participants
Tap to return to call

Currently looking into this